src/js/translations.js

```diff
@@ -1,19 +1,22 @@
 import en from '../translations/en.json';
 import fi from '../translations/fi.json';
 import de from '../translations/de.json';
+import es from '../translations/es.json';
 
 const languageSelector = document.getElementById('language-selector');
 
 const languageMap = {
   fi,
   en,
   de,
+  es,
 };
 
 export const languages = {
   'en': 'English',
   'fi': 'Suomi (Finnish)',
   'de': 'Deutsch (German)',
+  'en': 'Spanish (Spain)',
```

Comment:
this should be `Español (Spanish)`

Reply:
I see, so "Deutsch (German)" should not be in English either, right?

Reply:
`Deutsch` is German for German, so it's correct. the format is supposed to be `name of the original language (name of the language in english)`